app/src/components/layout/LogoComponents/ConfigurableLogo.tsx

```diff
@@ -5,32 +5,41 @@ import { loadConfig } from 'src/utils';
 import { Link } from 'react-router-dom';
 import { pxToRem } from 'src/styled-theme';
 import { useAppWidth } from 'src/hooks';
+import { getIsFirstVisit, useAppSelector } from 'src/store';
 
 export const ConfigurableLogo: React.FC = () => {
-  const { logoUrl } = loadConfig();
+  const { logoUrl, logoSize } = loadConfig();
```

Comment:
I don't know that we really want this to be part of the component. Do you know why it was there in the first place?

Reply:
This seemed like the most simple and direct way to handle new logos passed via .env file.
Is there a better approach?

Reply:
Yeah the code is right, but you should invoke this function outside of the component so it only runs once. I think most of the other `loadConfig` instances are outside of the component.

Reply:
➕

Reply:
+1